pkgs/top-level/python-packages.nix

```diff
@@ -7828,6 +7828,8 @@ in {
 
   userpath = callPackage ../development/python-modules/userpath { };
 
+  pipx = callPackage ../development/python-modules/pipx { };
+
```

Comment:
please attempt to sort it, this prevent collisions with other PRs which do the same, and it makes the status of `python-modules` a little more readible